op-challenger/game/keccak/matrix/testdata/commitments.json

```diff
@@ -0,0 +1,852 @@
+[
+  {
+    "input": "kYbKyRm02RR58P+t8WPE/UNL4zmr4XIe3qfMfFwRZVWlpzgDSmAXGNhqCGJZivDoNDRM1zrLaPAk5Ol8dAyi3shQgLHaBur7TLczzwxS9b8rU7WKPhTMsK+zblk3ps19yKixxPMjzMA5L70JOy0ng7/ZoJJhMuHC7TD1SYL5LEA=",
+    "commitments": [
+      "0xee0a1a26c607ab52c6308165995365f7951a185fccca4b76c847b8860d9fea7a",
+      "0x87d8548924b20b95fe1ed923236c44d2f98b8dc7a3c8c378e4193cc2745b1e8d"
+    ]
+  },
+  {
+    "input": null,
+    "commitments": [
+      "0xee0a1a26c607ab52c6308165995365f7951a185fccca4b76c847b8860d9fea7a"
+    ]
+  },
```

Comment:
The empty data case is particularly interesting.  Should there be one commitment with just the initial state or should we add a single leave that is purely padding and then having a second commitment after that?

Reply:
Looking at the contract, it seems like you have to call `addLeavesLPP` at least once though it's pretty academic because it will fail with `NotInitialized` if the claim size is 0 so you can' use this system to upload a zero length preimage which makes sense given its for large preimages.

Reply:
I think it technically should include padding. But this doesn't matter for our usecase as you've noted.

Reply:
Yeah the padding includes a domain separator and will definitely be added before the final hash is computed so I think it makes sense to include the padding in the final state commitment.